packages/taiko-client/cmd/flags/driver.go

```diff
@@ -52,6 +53,33 @@ var (
 		Category: driverCategory,
 		EnvVars:  []string{"BLOB_SOCIAL_SCAN_ENDPOINT"},
 	}
+	// preconf block server
+	PreconfBlockServerPort = &cli.Uint64Flag{
+		Name:     "preconfirmation.serverPort",
+		Usage:    "HTTP port of the preconfirmation block server, 0 means disabled",
+		Category: driverCategory,
+		EnvVars:  []string{"PRECONFIRMATION_SERVER_PORT"},
+	}
+	PreconfBlockServerJWTSecret = &cli.StringFlag{
+		Name:     "preconfirmation.jwtSecret",
+		Usage:    "Path to a JWT secret to use for the preconfirmation block server",
+		Category: driverCategory,
+		EnvVars:  []string{"PRECONFIRMATION_SERVER_JWT_SECRET"},
+	}
+	PreconfBlockServerCORSOrigins = &cli.StringFlag{
+		Name:     "preconfirmation.corsOrigins",
+		Usage:    "CORS Origins settings for the preconfirmation block server",
+		Category: driverCategory,
+		Value:    "*",
+		EnvVars:  []string{"PRECONFIRMATION_SERVER_CORS_ORIGINS"},
+	}
+	PreconfBlockServerCheckSig = &cli.BoolFlag{
```

Comment:
This flag can be decided by `PreconfBlockServerJWTSecret`.

Reply:
They are two different mechanisms